modules/travel_pay/app/controllers/travel_pay/claims_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class ClaimsController < ApplicationController
+    def index
+      veis_token = client.request_veis_token
+
+      # Non-intuitive Ruby behavior: #split splits a string on space by default
+      vagov_token = request.headers['Authorization'].split[1]
+      btsss_token = client.request_btsss_token(veis_token, vagov_token)
+
```

Comment:
Could you potentially refactor the request veis and btsss token into a method on the client class? Then you could call that from within the get_claims method.

Reply:
Yep! Good call, you read our minds. We have a [preemptive tech debt ticket](https://app.zenhub.com/workspaces/beneficiary-travel-btsss-65147e21930cd900223d8e64/issues/gh/department-of-veterans-affairs/va.gov-team/78913) in to change the design slightly to a) make our tests more robust and b) remove some of the logic from the controller.

Our thoughts are that we'll tackle that at a later date (hopefully soonish). What are your thoughts on if that needs to be done before we merge?